src/internal/log/field_test.go

```diff
@@ -54,4 +57,17 @@ func TestProto(t *testing.T) {
 
 	Info(ctx, "time", Proto("time", timestamppb.Now()))
 	Info(ctx, "nil time", Proto("time", (*timestamppb.Timestamp)(nil)))
+
+	v := &versionpb.Version{}
+	dynamic := dynamicpb.NewMessage(v.ProtoReflect().Descriptor())
+	dynamic.Set(v.ProtoReflect().Descriptor().Fields().ByName("major"), protoreflect.ValueOf(uint32(3)))
```

Comment:
Where does this 3 come from?

Reply:
Oh, just some random number created for testing.  `2` is too predictable for that field.